web/src/utils/flows.ts

```diff
@@ -35,24 +35,47 @@ const getInvolvedInterfaces = (flowsFor5Tuples: Record[]): { ifnames: string[];
   return { ifnames, ifdirs };
 };
 
+const getInvolvedUdns = (flowsFor5Tuples: Record[]): { udns: string[] } => {
+  const cache = new Set();
+  const udns: string[] = [];
+  flowsFor5Tuples.forEach(f => {
+    if (f.fields.Udns) {
+      _.zip(f.fields.Udns).forEach(([udn]) => {
+        const key = `${udn}`;
+        if (!cache.has(key)) {
+          cache.add(key);
+          udns.push(udn!);
+        }
+      });
+    }
+  });
+  return { udns };
+};
```

Comment:
```suggestion
const getInvolvedUdns = (flowsFor5Tuples: Record[]): string[] => {
  const cache = new Set();
  const udns: string[] = [];
  flowsFor5Tuples.forEach(f => {
    if (f.fields.Udns) {
      _.zip(f.fields.Udns).forEach(([udn]) => {
        const key = `${udn}`;
        if (!cache.has(key)) {
          cache.add(key);
          udns.push(udn!);
        }
      });
    }
  });
  return udns;
};
```

You just need to return an array of string here. Interfaces was caching both `ifnames` & `ifdirs` in a single object